exasol_script_languages_container_ci/lib/ci_build.py

```diff
@@ -1,29 +1,51 @@
-             rebuild: bool,
-             build_docker_repository: str,
-             commit_sha: str,
-             docker_user: str,
-             docker_password: str):
-    """
-    Build the script-language container for given flavor. And also build the test container
-    """
+class CIBuild:
+    def build(self,
+              flavor_path: Tuple[str, ...],
+              rebuild: bool,
+              build_docker_repository: Optional[str],
+              commit_sha: str,
+              docker_user: str,
+              docker_password: str,
+              test_container_folder: str) -> Tuple[Dict[str, Dict[str, ImageInfo]], ImageInfo]:
+        """
+        Build the script-language container for given flavor. And also build the test container
+        """
 
-    logging.info(f"Running command 'build' with parameters: {locals()}")
-    ctx.invoke(build, flavor_path=flavor_path, force_rebuild=rebuild,
-               source_docker_repository_name=build_docker_repository,
-               source_docker_username=docker_user, source_docker_tag_prefix=commit_sha,
-               source_docker_password=docker_password, shortcut_build=False, workers=7)
-    logging.info(f"Running command 'build_test_container' with parameters: {locals()}")
-    ctx.invoke(build_test_container, force_rebuild=rebuild, workers=7)
-    print_docker_images(logging.info)
+        logging.info(f"Running command 'build' with parameters: {locals()}")
+        if build_docker_repository is None:
+            slc_image_infos = build(flavor_path=flavor_path, force_rebuild=rebuild,
+                                    source_docker_tag_prefix=commit_sha,
+                                    source_docker_username=docker_user,
+                                    source_docker_password=docker_password,
+                                    shortcut_build=False,
+                                    workers=7)
+        else:
+            slc_image_infos = build(flavor_path=flavor_path, force_rebuild=rebuild,
+                                    source_docker_repository_name=build_docker_repository,
+                                    source_docker_tag_prefix=commit_sha,
+                                    source_docker_username=docker_user,
+                                    source_docker_password=docker_password,
+                                    shortcut_build=False,
+                                    workers=7)
+        logging.info(f"Running command 'build_test_container' with parameters: {locals()}")
+        content = build_test_container_content(test_container_folder)
+        test_container_image_infos = build_test_container(
+            force_rebuild=rebuild,
+            workers=7,
+            test_container_content=content,
+            source_docker_repository_name=build_docker_repository,
+            source_docker_tag_prefix=commit_sha
+        )
+        print_docker_images(logging.info)
+        return slc_image_infos, test_container_image_infos
```

Comment:
```suggestion
        logging.info(f"Running command 'build' with parameters: {locals()}")
        kwargs = {
            "flavor_path": flavor_path, 
            "force_rebuild": rebuild,
            "source_docker_tag_prefix": commit_sha,
            "source_docker_username": docker_user,
            "source_docker_password": docker_password,
            "shortcut_build": False,
            "workers": 7
        }

        if is not build_docker_repository:
            kwargs["source_docker_repository_name"] = build_docker_repository

        build(**kwargs)
        logging.info(f"Running command 'build_test_container' with parameters: {locals()}")

        content = build_test_container_content(test_container_folder)
        kwargs = { 
            k: v for k, v in kwargs.items() 
            if k in {'force_rebuild', 'workers', 'commit_sha'}
        )
        kwargs['test_container_content'] = content
        kwargs['source_docker_repository_name'] = build_docker_repository
        test_container_image_infos = build_test_container(**kwargs)
        print_docker_images(logging.info)

        return slc_image_infos, test_container_image_infos
```

Reply:
because we want to check the API, I would like to use syntax checking here